resources/definitions/nwa3d_a5.def.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "NWA3D A5",
+    "version": 1,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "DragonJe",
+        "manufacturer": "NWA 3D LLC",
+        "file_formats": "text/x-gcode",
+        "platform_texture": "nwa3d_backplate.png",
+        "platform_offset": [0, 0, 0],
+        "has_materials": true,
+        "has_variants": false,
+        "has_machine_materials": true,
+        "has_variant_materials": false,
+        "preferred_quality_type": "normal",
+        "has_machine_quality": true,
+        "preferred_material": "generic_pla",
+        "machine_extruder_trains": 
+        {
+            "0": "nwa3d_a5_extruder_0"
+        },
+    },
+  
+    "overrides": {
+        "machine_name": { 
+            "default_value": "NWA3D A5" 
+        },
+        "machine_width": { 
+            "default_value": 125 
+        },
+        "machine_height": { 
+            "default_value": 100 
+        },
+        "machine_depth": { 
+            "default_value": 150 
+        },
+        "machine_head_polygon": { 
+            "default_value": [
+                [-30, 34], 
+                [-30, -32], 
+                [30, -32], 
+                [30, 34]
+            ] 
+        },
+        "gantry_height": { 
+            "default_value": 30 
+        },
+        "machine_heated_bed": { 
+            "default_value": false 
+        },
+        "material_diameter": {
+            "default_value": 1.75
+        },
+        "machine_gcode_flavor": { 
+            "default_value": "RepRap (RepRap)" 
+        },
+        "machine_start_gcode": { 
+            "G28 ; Home\nG1 Z15.0 F6000 ; Move Z axis up 15mm\n ; Prime the extruder\nG92 E0\nG1 F200 E3\nG92 E0" 
```

Missing `default_value`
```suggestion
            "default_value": "G28 ; Home\nG1 Z15.0 F6000 ; Move Z axis up 15mm\n ; Prime the extruder\nG92 E0\nG1 F200 E3\nG92 E0" 
```